### ISSUE TITLE
Ankle instability

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11169]
**Preferred term label:**
Ankle instability

**Synonyms**

**Definition (free text, please give PubMed ID)**
Subjective or objective feeling of clunking/clicking in the ankle with movement, ankle 'giving way' or sensation of ankle dislocating

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Peroneal tendon disorder: https://www.ncbi.nlm.nih.gov/books/NBK544354/
 Anterior tibialis tendon rupture

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
The current term [Increased laxity of ankles HP:0006460](https://hpo.jax.org/browse/term/HP:0006460) is not well defined, but it seems very similar to Ankle instability (e.g., from the link above "Lateral ankle instability can cause laxity, leading to the increased motion of the tendons around the fibula with stretched superior peroneal retinaculum"), and I wonder if we should just use this term and provide a better definition? Or do you see a distinction?

**Comment by MickeySegal** [comment]
Laxity might be interpreted as ligamentous laxity, which may be part of the ankle instability.  At the minimum I'd add ankle instability to the entry.

**Comment by pnrobinson** [comment]
@MickeySegal  Coming back to this, I think that the existing term Increased laxity of ankles HP:0006460 is meant to descrine Ankle hypermobility, that can be a feature of Ehlers Danlos, or Ullrich congenital muscular dystrophy. Ankle instability is distinct. I will update definitions and create a separate term.

**Comment by pnrobinson** [comment]
@MickeySegal 
new term:
HP:0020309
Label: Ankle instability
Def; Failure of ligamentous and muscular structures of the ankle to maintain normal joint alignment during movement, resulting in excessive motion that causes manifestations such as subjective or objective feeling of clunking/clicking in the ankle with movement, ankle 'giving way' or sensation of ankle dislocating.
